src/plugin/node.js

```diff
@@ -0,0 +1,53 @@
+        'node/no-unsupported-features/es-syntax': 'error',           // disallow unsupported ECMAScript syntax on the specified version
+        'node/no-unsupported-features/node-builtins': 'error',       // disallow unsupported Node.js built-in APIs on the specified version
+        'node/process-exit-as-throw': 'off',                         // make process.exit() expressions the same code path as throw
+        'node/shebang': 'error',                                     // suggest correct usage of shebang
+
+        // Best Practices
+        'node/no-deprecated-api': 'error',                           // disallow deprecated APIs
+
+        // Stylistic Issues
+        'node/callback-return': 'off',                               // require return statements after callbacks
+        'node/exports-style': ['error', 'module.exports'],           // enforce either module.exports or exports
+        'node/file-extension-in-import': ['error', 'never'],         // enforce the style of file extensions in import/export declarations
+        'node/global-require': 'error',                              // require require() calls to be placed at top-level module scope
+        'node/no-mixed-requires': 'error',                           // disallow require calls to be mixed with regular variable declarations
+        'node/no-process-env': 'off',                                // disallow the use of process.env
+        'node/no-restricted-import': 'off',                          // disallow specified modules when loaded by import declarations
+        'node/no-restricted-require': 'off',                         // disallow specified modules when loaded by require
+        'node/no-sync': 'off',                                       // disallow synchronous methods
+        'node/prefer-global/buffer': ['error', 'always'],            // enforce either Buffer or require("buffer").Buffer
+        'node/prefer-global/console': ['error', 'always'],           // enforce either console or require("console")
+        'node/prefer-global/process': ['error', 'always'],           // enforce either process or require("process")
+        'node/prefer-global/text-decoder': ['error', 'always'],      // enforce either text-decoder or require("util").TextDecoder
+        'node/prefer-global/text-encoder': ['error', 'always'],      // enforce either text-encoder or require("util").TextEncoder
+        'node/prefer-global/url-search-params': ['error', 'always'], // enforce either URLSearchParams or require("url").URLSearchParams
+        'node/prefer-global/url': ['error', 'always'],               // enforce either URL or require("url").URL
+        'node/prefer-promises/dns': 'error',                         // disallow callback API in favor of promise API
+        'node/prefer-promises/fs': 'error',                          // disallow callback API in favor of promise API
+    },
```

Comment:
One of the major issues we want to solve with eslint update is `Stop linter complaining about optional chaining function execution, that doesn’t return, like  someObject?.someFunc?.()` (as described in the ticket) Is that already addressed?

Reply:
For context, atm for code like below we have to suppress the lint error, or do `void event?.preventDefault?.();`

```
export const preventDefaultThenAction = (actionName) => {
    const thenAction = action(`event: ${actionName}`);

    return (event) => {
        event?.preventDefault?.(); // eslint-disable-line no-unused-expressions

        thenAction();
    };
};
```

Reply:
franco I think that may have been resolved in [eslint 7.5](https://eslint.org/blog/2020/07/eslint-v7.5.0-released#optional-chaining-support)

so hopefully that means it'll go away automatically! perhaps give it a test @francox9

Reply:
yeah, this has been resolved by eslint, I no longer get that error when testing